asyncio_mqtt/client.py

```diff
@@ -300,12 +308,7 @@ async def force_disconnect(self) -> None:
     @_outgoing_call
     async def subscribe(
         self,
-        topic: (
-            str
-            | tuple[str, mqtt.SubscribeOptions]
-            | list[tuple[str, mqtt.SubscribeOptions]]
-            | list[tuple[str, int]]
-        ),
+        topic: SubscribeTopic,
```

Comment:
Good to see this compelx type definition moved out of the function definition. 👍